### PR TITLE
don't explicitly tag base:latest or cc:latest in cloudbuild.yaml

### PR DESCRIPTION
**Pull request by sharifelgamal**

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -1,24 +1,20 @@
 timeout: 1800s
-steps:  
+steps:
 - name: gcr.io/cloud-builders/bazel
   args: ['run', '//base:static']
 - name: gcr.io/cloud-builders/docker
   args: ['tag', 'bazel/base:static', 'gcr.io/$PROJECT_ID/static:latest']
 
 - name: gcr.io/cloud-builders/bazel
```
Comment (chanseokoh) [inline cloudbuild.yaml]: I wonder we can just remove `run //base:base` too. Unsure. For `run //cc:cc` too.

**Comment by chanseokoh** [comment]
Also, is it worth adding some comments whey they are missing? (If I did not have the background, I would have wondered why.)

File: cloudbuild.yaml [hunk 2]
```diff
@@ -106,9 +102,7 @@ steps:
 
 images:
   - 'gcr.io/$PROJECT_ID/static:latest'
```
Comment (chanseokoh) [inline cloudbuild.yaml]: Remove `static:latest` too, right?

Comment (sharifelgamal) [reply cloudbuild.yaml]: Yep. Fixed.

File: BUILD [hunk 3]
```diff
@@ -5,10 +5,7 @@ load("@io_bazel_rules_docker//container:container.bzl", "container_bundle")
 container_bundle(
     name = "all",
     images = {
-        "gcr.io/{PROJECT_ID}/go:latest": "//base:static",
```
Comment (chanseokoh) [inline BUILD]: Oh, actually this is not `static:latest` but `go:latest`. But probably this has to go anyway?

Comment (sharifelgamal) [reply BUILD]: We don't actually maintain anything called go anymore, the name change to static happened awhile ago.

**Pull request merged**